src/App/mermaidData/syncApiDataIntoOfflineStorage/SyncApiDataIntoOfflineStorage.js

```diff
@@ -170,12 +187,31 @@ const SyncApiDataIntoOfflineStorage = class {
             },
           )
           .then((response) => {
-            const projectsWithSyncErrors =
-              this.#getTableNamesWhereSyncingWasRejectedListedPerProjectFromSyncPushError(response)
-            const areThereSyncErrors = Object.keys(projectsWithSyncErrors).length
```

Comment:
woah, my code for handling 403s was named way too generically

Reply:
`allSyncStatusCodes` is good and clear